docs/DeveloperGuide.md

```diff
@@ -154,6 +154,18 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### tasksOn command
```

Comment:
I think it would be better to change this to "Viewing tasks on a particular day feature", which also means that you can probably include the `patientsToday` command under this section since it is a more specific version of `taskOn`.

Reply:
changed